Tests/Update-NFS.Tests.ps1

```diff
@@ -19,7 +19,7 @@ Process {
         $nfsadvconfig.Values | ForEach-Object -Process {
             $compare += $_
         }
-        foreach ($server in (Get-VMHost -Name $config.scope.host).name) 

```

Comment:
I think that `.name` is either still needed here, or the code below needs to be updated.